<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>236</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>543</width>
    <height>236</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>543</width>
    <height>236</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Moyene</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="userInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="instructionLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Veuillez entrer un nombre s'il vous plait!!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resultatLabel_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="toolTipDuration">
     <number>5</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Programme pour calculer la moyenne d'une suite de nombre</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="addNumber">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter un nombre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="averageButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Moyenne</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
